packages/xo-web/src/common/xo/index.js

```diff
@@ -1380,10 +1380,17 @@ export const hidePcis = async (pcis, hide) => {
       // hide `true` means that we will disable dom0's PCI access, so we will "enable" the possibility of passthrough this PCI
       title: _(hide ? 'pcisEnable' : 'pcisDisable', { nPcis: pcis.length }),
     })
+
+    await _call('pci.disableDom0Access', { pcis: resolveIds(pcis), disable: hide })
   } catch (error) {
-    return
+    if (noHostsAvailable.is(error)) {
+      await confirm({
+        body: _('hostEvacuationFailed'),
+        title: _('confirmForceRebootHost'),
+      })
+      await _call('pci.disableDom0Access', { pcis: resolveIds(pcis), disable: hide, force: true })
```

Comment:
This silently swallows any other type of error. See [my comment](https://github.com/vatesfr/xen-orchestra/pull/7687#discussion_r1616780774) to rethrow the error.